tests/wdio/home-spec.js

```diff
@@ -0,0 +1,19 @@
+Terra.describeViewports('home', ['tiny', 'huge'], () => {
+  before(() => {
+    const viewport = browser.getViewportSize();
+    viewport.height = 2100;
+    browser.setViewportSize(viewport);
+  });
+
+  it('checks accessibility', () => {
+    browser.url('/');
+    // browser.pause(1500);
+    browser.waitForVisible('a [alt="devDependencies status"]');
+    browser.waitForVisible('a [alt="Dependencies status"]');
+    browser.waitForVisible('a [alt="Build Status"]');
+    browser.waitForVisible('a [alt="License"]');
+    browser.waitForVisible('a [alt="Cerner OSS"]');
+    browser.waitForVisible('a [alt="NPM version"]');
+    Terra.validates.accessibility({ selector: '#root' });
+  });
```

Comment:
Looks like there is a stale home-spec screenshot but we are no longer taking a screenshot in this test spec. Needs removed: https://github.com/cerner/terra-dev-site/blob/8e03dc91f6e128d1fdedc2f2ae4bfb29647f1416/tests/wdio/__snapshots__/reference/en/chrome_tiny/home-spec/home%5Bdefault%5D.png